Orange/widgets/gui.py

```diff
@@ -3240,3 +3241,16 @@ def setRange(self, minValue, maxValue, step=1.0):
         # For compatibility with qwtSlider
         # TODO If it's related to Qwt, remove it
         self.setScale(minValue, maxValue, step)
+
+
+class ControlGetter:
+    def __init__(self, widget):
```

Comment:
This class could use a docstring describing what it does and how does it actually find controls.